### PR TITLE
Add glyph references for Cupertino-style icons

### PR DESCRIPTION
**Pull request by ndrong**
- add 11 glyph references
- add comments to similar glyphs to reference each other

This adds a few of the missing glyph references, as discussed in issue #16102.

### CODE REVIEW

**Comment by NDrong** [comment]
I signed it!

File: packages/flutter/lib/src/cupertino/icons.dart [hunk 1]
```diff
@@ -99,4 +103,41 @@ class CupertinoIcons {
...
+
+  /// A curved up and left pointing arrow.
+  ///
+  /// For an outlined version of this icon, see [reply].
+  static const IconData reply_filled = const IconData(0xf21e, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Old (pre-iOS 7) iOS style share icon with an arrow pointing to the right from a box.
+  ///
+  /// For a newer version of this icon (introduced in iOS 7), see [share].
+  static const IconData share_old = const IconData(0xf220, fontFamily: iconFont, fontPackage: iconFontPackage);
```
Comment (xster) [inline packages/flutter/lib/src/cupertino/icons.dart]: Don't use relative terms. Everything will become old at some point. Call it share_pre_7 or something

File: packages/flutter/lib/src/cupertino/icons.dart [hunk 2]
```diff
@@ -99,4 +103,41 @@ class CupertinoIcons {
...
+
+  /// Two right-facing intertwined arrows.
+  static const IconData merge_arrow = const IconData(0xf221, fontFamily: iconFont, fontPackage: iconFontPackage);
```
Comment (xster) [inline packages/flutter/lib/src/cupertino/icons.dart]: They're not really merging. I think it's the shuffle symbol. There's also other ones at F4A8. I'd call this 'shuffle_thick' or something

File: packages/flutter/lib/src/cupertino/icons.dart [hunk 3]
```diff
@@ -99,4 +103,41 @@ class CupertinoIcons {
...
+  /// A rewind arrow, pointing downwards.
+  static const IconData rewind_arrow = const IconData(0xf21c, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// Two curved up and left pointing arrows.
+  static const IconData reply_all = const IconData(0xf21d, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A curved up and left pointing arrow.
+  ///
+  /// For an outlined version of this icon, see [reply].
+  static const IconData reply_filled = const IconData(0xf21e, fontFamily: iconFont, fontPackage: iconFontPackage);
```
Comment (xster) [inline packages/flutter/lib/src/cupertino/icons.dart]: If F4C6 was already `reply`, I'd expect F4C7 which has a similar style to be `reply_solid`. Maybe this should be `reply_thick_solid`

File: packages/flutter/lib/src/cupertino/icons.dart [hunk 4]
```diff
@@ -99,4 +103,41 @@ class CupertinoIcons {
...
+  static const IconData battery_75_percent = const IconData(0xf114, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A 25% charged battery.
+  static const IconData battery_25_percent = const IconData(0xf115, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// The bluetooth logo.
+  static const IconData bluetooth = const IconData(0xf116, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A rewind arrow, pointing downwards.
+  static const IconData rewind_arrow = const IconData(0xf21c, fontFamily: iconFont, fontPackage: iconFontPackage);
```
Comment (xster) [inline packages/flutter/lib/src/cupertino/icons.dart]: I was expecting F4A0 to be the rewind icon. This is probably `restart` or something

File: packages/flutter/lib/src/cupertino/icons.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter/lib/src/cupertino/icons.dart]: Maybe even `share_up`?

**Comment by NDrong** [comment]
@xster Thanks for the feedback! I've updated the references per your comments.

**Comment by xster** [comment]
Cool, thanks for making the changes. Merging

**Comment by zoechi** [comment]
Related to #16102

**Pull request merged**